astropy/io/ascii/fixedwidth.py

```diff
@@ -71,7 +71,7 @@ class FixedWidthHeader(basic.BasicHeader):
     """ Splitter class for splitting data lines into columns """
     position_line = None  # secondary header line position
     """ row index of line that specifies position (default = 1) """
-    set_of_position_line_characters = set(r'`~!#$%^&*-_+=\|":' + "'")
+    set_of_position_line_characters = set(r'`~!#$%^&*-_+=\|":' "'")
```

Comment:
This change looks like it might conflict with [ISC001](https://beta.ruff.rs/docs/rules/single-line-implicit-string-concatenation/).

Reply:
Agreed on the conflict with implicit string concatenation. But more importantly here the original is really preferred due to the high density of special characters.

Reply:
would 
```suggestion
    set_of_position_line_characters = set(r"""`~!#$%^&*-_+=\|":'""")
```
work?

Reply:
Good idea, fine by me!